src/System.Private.ServiceModel/src/System/ServiceModel/Channels/RTHttpClientHandler.cs

```diff
@@ -0,0 +1,475 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Actually the naming convention doesn't imply it is a partial class (that I know).  It just means "this file contains code only for XXX".  I think we need to choose some convention for files that are strictly for one OS or target group, regardless how they are structured inside.  I thought the new naming convention was a reasonable proposal for that.  The fact we include "**\*.cs" makes it especially non-obvious which of our files are OS/target specific.